src/registrar/tests/common.py

```diff
@@ -2,13 +2,20 @@
 import logging
 
 from contextlib import contextmanager
+import random
+from string import ascii_uppercase
 from unittest.mock import Mock
 from typing import List, Dict
 
 from django.conf import settings
 from django.contrib.auth import get_user_model, login
 
-from registrar.models import Contact, DraftDomain, Website, DomainApplication, User
+from registrar.models import Contact, DraftDomain, Website, DomainApplication
```

Comment:
```suggestion
from registrar.models import Contact, DraftDomain, Website, DomainApplication, DomainInvitation, User, DomainInformation, Domain
```

remove repeated import and just add to the existing one

Reply:
Good catch, not sure how that snuck in